stdlib/3/os/__init__.pyi

```diff
@@ -359,11 +359,9 @@ def symlink(source: _PathType, link_name: _PathType,
             target_is_directory: bool = ...) -> None:
     ...  # final argument in Windows only
 def unlink(path: _PathType) -> None: ...
-# TODO: add ns, dir_fd, follow_symlinks argument
-if sys.version_info >= (3, 0):
```

Comment:
Could you put this back? I added it so that we can eventually merge the Python 2 and 3 versions of os.

Reply:
Will do.